cypher-shell/src/test/java/org/neo4j/shell/cli/InteractiveShellRunnerTest.java

```diff
@@ -123,27 +112,22 @@ public void runUntilEndShouldStopOnExitExceptionAndReturnCode() throws IOExcepti
     }
 
     @Test
-    public void ctrlCDoesNotKillInteractiveShell() throws Exception {
+    public void ctrlCKillsInteractiveShell() throws Exception {
         String input = "good1;\n" +
                 "good2;\n" +
                 ctrl('C') +
```

Comment:
NO. This doesn't do anything anymore. I created the PR before I spoke to you about the test. I have updated the PR